scripts/ssl-manager.js

```diff
@@ -475,8 +475,16 @@ function SSLManager(config) {
     };
 
     me.initAddOnExtIp = function initAddOnExtIp(withExtIp) {
-        config.withExtIp = (typeof withExtIp === "boolean") ? !!withExtIp : !!(withExtIp == "true");
-
+        if (!jelastic.env.binder.GetExtDomains) {
+            config.withExtIp = true;
+            return { result: 0 };
+        }
+        
+        if (typeof withExtIp == "boolean") {
+            config.withExtIp = withExtIp;
+        } else {
+            config.withExtIp = String(withExtIp) != "false");
+        }
```

Comment:
```
config.withExtIp = ((typeof withExtIp == "boolean") ? withExtIp : String(withExtIp) != "false")
    || !jelastic.env.binder.GetExtDomains;
```
should be better

Reply:
done - https://github.com/dzotic9/lets-encrypt/commit/0e46153597f3045307729706d991c737f2cac9cb